tools/mapset/map.go

```diff
@@ -41,10 +39,6 @@ func (m *MapSet) Reserve(key string, value string) (result Result) {
 		m.mapOfSet[key] = innerSet
 	}
 
-	if m.mapOfSet[key].Has(value) {
-		result.Error = ErrAlreadyExists
```

Comment:
Per the description of the method, it sounds like this should return `false` but _not_ return an error.